.gitignore

```diff
@@ -1,7 +1,8 @@
 config.toml
 dcrstakepool
-.vendor/
+vendor/
```

Comment:
I believe this line could be removed entirely

Reply:
I use the vendor folder locally within my IDE to enable me explore source code for external packages easily. Doesn't hurt to keep it, lest I or someone else mistakenly checks their vendor folder into a PR.